modules/tensor_mechanics/include/materials/ComputeStressEosBase.h

```diff
@@ -0,0 +1,47 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef COMPUTESTRESSEOSBASE_H
+#define COMPUTESTRESSEOSBASE_H
+
+#include "ComputeStressBase.h"
+
+/**
+ * ComputeStressEosBase adds a volumetric extra stress including a Birch-Murnaghan EOS
```

Comment:
Include the citation information if you are implementing equations from a specific paper (e.g. Birch Physical Review 71 (1947) 809-824)